app/main/views/suppliers.py

```diff
@@ -850,6 +850,10 @@ def toggle_supplier_services(supplier_id):
     if not toggle_action['framework_slug']:
```

Comment:
This is actually about the line a few up from here - what happens if both `remove_services` and `publish_services` are truthy and are _different_ framework slugs?

Reply:
I doubt we thought ahead that far. I think it will default to the `remove` action if both are supplied.